biodata-models/src/main/java/org/opencb/biodata/models/variant/stats/VariantStats.java

```diff
@@ -426,15 +426,23 @@ public VariantStats calculate(Map<String, Map<String, String>> samplesData, Map<
                     if (g.getAllele(0) < 0) {
                         this.setMissingAlleles(this.getMissingAlleles() + 1);
                     } else {
-                        allelesCount[g.getAllele(0)]++;
-                        totalAllelesCount++;
+                        //Allele counts for multiple alternates not supported by biodata!
+                        //Skip allele counts for genotypes like 2/-1 (getAllele(0) returns 2)
+                        if (g.getAllele(0) <= 1) {
```

Comment:
It means the same but I find this clearer. If you change `g.getAllele(0)` to something like `firstAllele` even better
```suggestion
                        if (g.getAllele(0) == 0 || g.getAllele(0) == 1) {
```

Reply:
Fair point. Addressed!